conteudo/exercicios/7-exercicio-projeto-casa/index.html

```diff
@@ -239,13 +239,14 @@ <h3>Alunas</h3>
           <p>Valéria Nascimento de Jesus</p>
       </div>
       <div class="container__aluna">
-          <a href="#" target="_blank">
-            <img class="container__aluna-img" src="https://via.placeholder.com/500" alt="">
+          <a href="#https://github.com/vaosorio" target="_blank">
```

Comment:
Oi Vanessa, tudo bom ?
Aqui foi com o # na frente do link, isso faz com que ele não vá pro seu perfil, adicionei a sugestão aqui:
```suggestion
          <a href="https://github.com/vaosorio" target="_blank">
```

Essa modificação não influencia na sua presença, já marquei como entregue :purple_heart: , é só para seu perfil ficar clicável e direcionar pra sua conta git corretamente

Reply:
Lydia, aceitei a sugestão de commit, preciso dar o push novamente?

Reply:
Precisa não ! Aprovado aqui, brigadão.